packages/governance/src/contractGovernor.js

```diff
@@ -281,6 +282,7 @@ const start = async (zcf, privateArgs) => {
     voteOnApiInvocation,
     voteOnOfferFilter: voteOnFilter,
     getCreatorFacet: () => limitedCreatorFacet,
+    getAdminFacet: () => adminFacet,
```

Comment:
This gave me some pause... we talked it over with @erights and agreed it's appropriate.